README.md

```diff
@@ -146,7 +146,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 ---
 
 - ### **A**
-
+  - [Anshu Pathak](https://github.com/AnshuPathak-88825)
```

Comment:
Please add your name in line 254, after Ansh Dhingra.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix